### PR TITLE
Fix German Localization typos/errors

### PR DESCRIPTION
**Pull request by TheStachelfisch**
Just fixed some typos and overall not grammar correct Sentences

### CODE REVIEW

File: patches/Terraria/Terraria.Localization.Content.de-DE.Main.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		"33": "Terraria: Bald auf einem Computer in deiner Nähe",
 		"34": "Terraria: Teilen durch Null",
 		"35": "Terraria: Jetzt mit SOUND",
-		"36": "Terraria: Drücke Alt-F4",
```
Comment (direwolf420) [inline patches/Terraria/Terraria.Localization.Content.de-DE.Main.json:33]: Drücke is correct

File: patches/Terraria/Terraria.Localization.Content.de-DE.Main.json [hunk 2]
```diff
@@ -156,15 +156,15 @@
...
-		"MASTERMIND_Description": "Besiege das Gehirn Cthulhus, ein riesiges Dämonenhirn, das das kriechende Purpur heimsucht.",
+		"MASTERMIND_Description": "Besiege das Gehirn des Cthulhus, ein riesiges Dämonenhirn, das das kriechende Purpur heimsucht.",
 		"MASTERMIND_Name": "Superhirn",
 		"MATCHING_ATTIRE_Description": "Rüste in allen drei Rüstungsslots Rüstung aus: Kopf, Brust, Füße.",
 		"MATCHING_ATTIRE_Name": "Passende Kleidung",
 		"MECHA_MAYHEM_Description": "Kämpfe gegen die Zwillinge, den Zerstörer und Skeletron Prime gleichzeitig und besiege sie.",
 		"MECHA_MAYHEM_Name": "Mecha Mayhem",
 		"MINER_FOR_FIRE_Description": "Stelle eine geschmolzene Spitzhacke her, indem du nur die heißesten Materialien benutzt.",
 		"MINER_FOR_FIRE_Name": "Feuerbergarbeiter",
-		"NoCategory": "Keine",
```
Comment (direwolf420) [inline patches/Terraria/Terraria.Localization.Content.de-DE.Main.json:167]: Kategorie, not Katerogie

File: patches/Terraria/Terraria.Localization.Content.de-DE.Main.json [hunk 3]
```diff
@@ -603,7 +603,7 @@
 		"WorldSizeUnknown": "Unbekannt",
 		"BartenderHelp": "Eternia-Kristall",
 		"CopySeed": "Seed kopieren: {0}",
-		"EnterSeed": "Gib Seed ein (für Zufallsseed freilassen)",
```
Comment (direwolf420) [inline patches/Terraria/Terraria.Localization.Content.de-DE.Main.json:606]: zufälligen not Zufälliggen

File: patches/Terraria/Terraria.Localization.Content.de-DE.Main.json [hunk 4]
```diff
@@ -665,15 +665,15 @@
...
+		"Ruptured": "{0}s lebensnotwendige Organe wurden zerfetzt.",
 		"SkullCrushed": "{0}s Schädel wurde eingeschlagen",
 		"Slain": "{0} wurde ermordet.",
 		"Snapped": "{0} wurde in zwei Teile geteilt.",
 		"TornInHalf": "{0} wurde in zwei Hälften gerissen."
 	},
 	"DungeonDefenders2": {
 		"BartenderWarning": "Der Eternia-Kristall sträubt sich gegen dieses Gebiet und teleportiert sich sofort weg. Der Tavernenwirt hatte erwähnt, dass er in einem großen, flachen und offenen Gebiet aufgestellt werden sollte ...",
-		"CantSummonTower": "It doesn't seem to work without an Etheria Crystal nearby...",
+		"CantSummonTower": "Es sieht nicht aus ohne einen Eternia-Kristall in der nähe zu funktionieren...",
```
Comment (direwolf420) [inline patches/Terraria/Terraria.Localization.Content.de-DE.Main.json:676]: I would write it like `Es scheint nicht ohne einen Eternia-Kristall in der nähe zu funktionieren...`

**Comment by TheStachelfisch** [comment]
> 
> 
> Some mistakes I found

Thanks, fixed those mistakes.

**Comment by JavidPack** [comment]
Thanks a lot, I had to take your changes and make a separate commit, since they were in the old location, but it all worked out.

**Pull request closed without merging**